src/physics/convection_diffusion.h

```diff
@@ -45,12 +47,14 @@ class ConvectionDiffusion : public PhysicsBase <dim, nstate, real>
         const dealii::Tensor<2,3,double>                          input_diffusion_tensor = Parameters::ManufacturedSolutionParam::get_default_diffusion_tensor(),
         const dealii::Tensor<1,3,double>                          input_advection_vector = Parameters::ManufacturedSolutionParam::get_default_advection_vector(),
         const double                                              input_diffusion_coefficient = Parameters::ManufacturedSolutionParam::get_default_diffusion_coefficient(),
-        std::shared_ptr< ManufacturedSolutionFunction<dim,real> > manufactured_solution_function = nullptr) : 
+        std::shared_ptr< ManufacturedSolutionFunction<dim,real> > manufactured_solution_function = nullptr,
+        const Parameters::AllParameters::TestType parameters_test = Parameters::AllParameters::TestType::run_control) : 
```

Comment:
if this parameter is being passed to all the derived class constructors, move it to the `PhysicsBase` constructor to reduce code repetition

Reply:
fixed as per issue https://github.com/dougshidong/PHiLiP/issues/126